cockatrice/src/settings/serverssettings.h

```diff
@@ -4,16 +4,23 @@
 #include "settingsmanager.h"
 #include <QObject>
 
+
 class ServersSettings : public SettingsManager
 {
     Q_OBJECT
     friend class SettingsCache;
 
-public:    
+public:
+    QStringList getDefaultHostList() {
+       return QStringList()
```

Comment:
Is this ok to keep in the header or should the function content be moved to the cpp file?